frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view_helpers.ts

```diff
@@ -97,31 +89,25 @@ export function insertTreesAndTransform(
       treeNode.children, // Groups that don't contain any trees should not influence the state of their parents
       (groupOrTree) => groupOrTree.isChecked || !groupOrTree.containsTrees,
     );
-    treeNode.isIndeterminate = treeNode.isChecked
```

Comment:
The `indeterminate` state is not no longer directly controlled by use. Antd takes care of that.